src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketTagSCMHead.java

```diff
@@ -39,44 +37,14 @@
 
     private static final long serialVersionUID = 1L;
 
-    /**
-     * Cache of the repository type.
-     *
-     * @since 2.2.11
-     */
-    // The repository type should be immutable for any SCMSource.
-    @CheckForNull
-    private final BitbucketRepositoryType repositoryType;
```

Comment:
BitbucketTagSCMHead has a `serialVersionUID` assigned so I presume it is serialized. Is that only for agent-controller communication or do these things get saved to files as well? Will the removal of the `repositoryType` field cause any problems for deserialization?

Reply:
Again, good catch. Didn't payed atention to serialization.

Reply:
I reduced performed changes